### ISSUE TITLE
Midsummer Festival issues with Honoring / Desecrating

### ISSUE THREAD

**Issue by goldstinger** [WoWManiaUK/Redemption#1312]
I can't honor, or desecrate any of the alliance or horde Midsummer Bonfires anywhere. I can get some of the quests fine and I've already killed Ahune multiple times, but I cannot interact outside of a dialogue box with Flame Keepers (on Horde), or desecrate (alliance) Bonfires.

Normally, there's some quick talk to & turn in quests according to the armory when you talk to them/visit them, but I get none of them. I've tried visiting locations in Durotar, Barrens, and both alliance & horde locations in Ashenvale, with no luck.

I'm thinking it might have something to do specifically with it being a wotlk quest while I'm playing a cata character race (goblin), but haven't heard from anyone else of it being an issue for them, so got no basis on that other than the 2 cents of my take on it.

Associated images:
http://i.imgur.com/9A13jfV.jpg
http://i.imgur.com/V7Hzwk8.jpg
http://i.imgur.com/gXbuqoj.jpg
http://i.imgur.com/2VjX7aJ.jpg
http://i.imgur.com/CPx9tcL.jpg
http://i.imgur.com/7t1dGP0.jpg

**Comment by squalllh1989** [comment]
To honor the flame keeper, you must do the quest line to the point there is  a quest that introdue you to honor one flame keeper, to get that quest, you must got pass the quest "Torch Catching" which is reported bugged here: https://github.com/WoWManiaUK/Redemption/issues/1307 , i was lucky that a gm response to my ticket and help me go throught that step so i can continue on the line.

**Comment by squalllh1989** [comment]
But, there is some trouble with the bon fire that at make it really hard to click on to desecrate, especially at Loch Modan[32,40] (Thelsamar ) and Bloodmyst Isle[55,69] (Blood Watch), you still can click on it, but you need to chose right place to click on, harder than it should be, and the two place that i listed here are the hardest when i have to change many position and angle to get a point that i can click on to desecrate  the fire. That's on the horde side, thought, i only play horde side, so i don't know if there is some trouble with the alliance side.

**Comment by goldstinger** [comment]
double checked on the PTR with a worgen and a night elf (same starting location), the level 1 night elf could honor the Bonfire, the level 1 worgen could not. Seems like a character race issue.

**Comment by squalllh1989** [comment]
oh, if it's like that then i think this is issue with the new race only, goblin and worgen.

**Comment by DarkestDuck** [comment]
Last year I had the same issue with goblin, but this was working for all my other (non-goblin) characters. I tried to open ingame ticket, but GM said it is OK and there are no such quests for me..

**Comment by fvicaria** [comment]
Is this an issue only for the new classes? Worgen/goblin.

**Comment by goldstinger** [comment]
Yep, only an issue for Cataclysm-related classes (Goblin & Worgen)

**Comment by fvicaria** [comment]
Can you send me an example of quests you could not get please?

**Comment by goldstinger** [comment]
http://www.wow-mania.com/armory/?quest=11841
http://www.wow-mania.com/armory/?quest=11581

Related section for both pages in the armory list duplicate quests for the different locations.

**Comment by HarryOger** [comment]
To update the extent of the issue: I have an Orc Hunter, lvl 80 who cannot interact with the Alliance flames at Loch Modan and Bloodmyst.  If there is and angle or position from which I have not tried these it must be microscopic.  If it helps, on both flames there appear to be at least 3 overlapping but offset bonfires in both of the above locations and these are the only 2 fires which appear this way. I have completed all other Horde and Alliance fires on this toon.

**Comment by shulasassi** [comment]
i have an ally 80, i honored and desecrated all bonfires except for desecrating the ones at Sepulcher, Stranglethorn Vale, and Stonard. a few took some monkeying around, trying to find that sweet spot was a challenge lol.

**Comment by fvicaria** [comment]
I have removed the duplicate objects.
